package.json

```diff
@@ -5,14 +5,14 @@
   "license": "MIT",
   "repository": "chalk/ansi-styles",
   "author": {
-    "name": "Sindre Sorhus",
     "email": "sindresorhus@gmail.com",
+    "name": "Sindre Sorhus",
```

Comment:
Why did you change this? Also let's not change unrelated things in a random commmit ;)